api/proto/teleport/legacy/types/types.proto

```diff
@@ -5627,6 +5565,9 @@ message OIDCAuthRequest {
   teleport.attestation.v1.AttestationStatement tls_attestation_statement = 23 [(gogoproto.jsontag) = "tls_attestation_statement,omitempty"];
   // pkce_verifier is used to verified a generated code challenge.
   string pkce_verifier = 24 [(gogoproto.jsontag) = "pkce_verifier"];
+  // login_hint is an optional username/email provided by the client that will be passed
```

Comment:
For the doc comment, you can format it like this:
```suggestion
  // LoginHint is an optional username/email provided by the client that will be passed
```

This is because even though the protobuf field is called `login_hint`, the auto-generated Go code will end up saying `LoginHint`.